src/App.js

```diff
@@ -1,24 +1,57 @@
-import logo from './logo.svg';
+/* import logo from './logo.svg'; */
 import './App.css';
+import allFoods from './foods.json';
+import { useState } from 'react'
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
 
 function App() {
+  
+  const [foods, setFoods] = useState(allFoods);
+  const [showFoods, setShowFoods] = useState(allFoods);
+
+  const createFood = (food) => {
+
+    const newFood = [food, ...foods];
+    setFoods(newFood)
```

Comment:
Every time you create a new food you need to update the `foods` like you did here. But since you have another `useState` for the result of the search you need to update it as well. So under the `setFoods(newFood)` you just need to add this line `setShowFoods(newFood)` same thing you did for the `deleteFood` function!